adafruit_rgb_display/__init__.py

```diff
@@ -1 +1,2 @@
+"""import class color565"""
```

Comment:
This string here is a bit confusing. First, it's a module-level docstring, so it should be describing the module, rather than commenting the code. Second, color565 is a function, not a class. And finally I'm not sure we need to explain what the "import" statement does.